features/handled.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Handled Exceptions are received
+
+Scenario: Notify is called
+  Given I configure the bugsnag endpoint
+  And I set environment variable "MAZE_API_KEY" to "1234567890"
+  When I run the console app "DotNetCore2Console" with "handled"
```

Comment:
This doesn't currently change the scenario if something different from 'handled' was set.

Reply:
It passes`handled` in as the args to the console application which uses it. I wanted to keep this PR small so the console application doesn't implement any other args yet. Once it does altering this will flow through

Reply:
Sounds good to me.